changelogs/fragments/6270-gconftool2-changed.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - gconftool2 - fix ``changed`` result always being ``true`` (https://github.com/ansible-collections/community.general/issues/6028).
```

IMO it's best to always also reference the PR which did the change:
```suggestion
  - gconftool2 - fix ``changed`` result always being ``true`` (https://github.com/ansible-collections/community.general/pull/6270, https://github.com/ansible-collections/community.general/issues/6028).
```